src/workerd/io/worker-interface.capnp

```diff
@@ -64,6 +65,10 @@ struct Trace @0x8e8d911203762d34 {
     }
   }
 
+  struct ConnectEventInfo {
+    cfJson @0 :Text;
```

Comment:
Not new in your change, but `cfJson` really ought to have been at the top level of `Trace`, since it applies to all event types... not sure if it's worth fixing though.